app/models/druid_version_zip.rb

```diff
@@ -137,8 +140,18 @@ def zip_storage
 
   private
 
+  # should throw an error if any of the files in the moab are not yet readable, for example due to
+  # Ceph MDS instance for a pres cat worker VM thinking that a file is a stray as a result of our
+  # particular use of hardlinking in preservation ingest.  allows for a quick directory walk before
+  # attempting to create the zip file(s).  see https://github.com/sul-dlss/preservation_catalog/issues/1633
+  # @raise [StandardError] if storage_location is not available (should have been provided in constructor)
+  # @raise [Errno::EACCES, ?] if it is not possible to stat one or more files in the Moab
```

Comment:
```suggestion
  # @raise [Errno::EACCES] if it is not possible to stat one or more files in the Moab
```

Reply:
So in this case, I was not entirely sure what the full list of plausible errors might be.  Admittedly, I didn't look back through all of the HB alerts when I first did this, but I wanted to call out that any number of ruby file read errors could be raised.

Looking back through Honeybadger alerts now, I also see some others that've occurred when reading Moab contents, which I'll added to the list, resulting in:  `Errno::EACCES, Errno::EIO, Errno::ENOENT, Errno::ESTALE`.

Still, looking over all the possibilities for error codes in `Errno.constants`, it's clear that most are not relevant to this method, but I suspect that some which we haven't seen are also plausible error codes.  For example, `EBUSY`, `ENOMEM`, and `EPERM`.

My bad for not being more thorough in listing the errors we've already encountered, but I would like to indicate that there are errors which might be raised, but which we're unsure about.  Though if you know for sure that the error codes I mentioned speculatively can't come up, happy to leave off the `?`.

i realize that the `?` may not technically be valid YARD syntax, but to my knowledge we don't auto-generate documentation off the YARD comments, and i though there was more value in expressing the uncertainty about possible errors than there was in producing strictly compliant YARD comments.

(I also just spent about 10 minutes, which was 10 minutes more than i should've, reading a 2003 Slashdot thread about some controversy with the `EDOOFUS` error code.  Same as it ever was 😆 )

Reply:
feel free to file an issue if you want me to circle back on this one again.